packages/react-router-config/package.json

```diff
@@ -21,8 +21,7 @@
     "watch": "babel ./modules -d . --ignore __tests__ --watch",
     "prepublishOnly": "node ./tools/build.js",
     "clean": "git clean -fdX .",
-    "lint": "eslint modules",
-    "test": "jest"
```

Comment:
Should we get rid of the test scripts in the individual packages? What if I want to watch on only one package?

Reply:
That is a good point. I don't know the full context of the project, so if this proposal does not align with the overall direction of the project, I totally understand 😄 

There are a couple of options
* Jest allows to filter using a regex `jest --watch react-router-dom`
* You can call Jest with only that project `jest --projects packages/react-router-dom`